src/tests.builds

```diff
@@ -3,6 +3,7 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
     <ExcludeProjects Condition="'$(OSEnvironment)'!='Windows_NT'" Include="**\Microsoft.VisualBasic.Tests.csproj" />
+    <ExcludeProjects Condition="'$(OSEnvironment)'=='Windows_NT'" Include="**\System.Security.Cryptography.OpenSsl.Tests.csproj" />
```

Comment:
How does this mechanism related to the ```<UnsupportedPlatforms/>``` mechanism that currently exists in the .csproj?

And if we list out Unix-specific test projects to be excluded on Windows, do we need to do the inverse and list out Windows-specific test projects to be excluded on Unix (e.g. Microsoft.Win32.Registry.Tests)?